TurkishFilter/sections/specific.txt

```diff
@@ -119,6 +119,8 @@ birhayalinpesinde.com##center > a[href*="://bit.ly/"] > img
 memurlar.net##.ColumnContainer > div[style="float:left; width:445px; height:350px"]
 ||memurlar.net/common/widget-image/kitapsec/16.jpg
 memurlar.net##body > #masth.PageWidth[style*="text-align:center!important"]
+doeda.com##center > p[style="background-color: #e7e7e7; color: #F00; font-weight: bold; font-size: 18px; text-align: center;"]
+||doeda.com/wp-content/*/porno.gif
```

Comment:
Link to the site itself, would move it to Annoyances. I am aware of the Sponsor text above it.

Reply:
agree